install.sh

```diff
@@ -120,17 +133,17 @@ fi
 echo "Will now create LXC Container $LXC_NBR!";
 
 # Create the container
-pct create $LXC_NBR -unprivileged $LXC_UNPRIVILEGED $LXC_TEMPLATE_STORAGE:vztmpl/"$LXC_TEMPLATE_VERSION"_$DEB_REP\_amd64.tar.gz -rootfs $LXC_ROOTFS_STORAGE:$LXC_ROOTFS_SIZE;
+pct create $LXC_NBR --password $LXC_PWD -unprivileged $LXC_UNPRIVILEGED "$LXC_TEMPLATE_STORAGE:vztmpl/${LXC_TEMPLATE_VERSION}_${DEB_REP}_amd64.tar.gz" -rootfs $LXC_ROOTFS_STORAGE:$LXC_ROOTFS_SIZE;
```

Comment:
Added lxc template variable, to generate template name. This conflicts with current code. I will add password parameter here manually.